src/hooks/useWaku.ts

```diff
@@ -28,16 +28,16 @@ export const useWaku = () => {
       setMessages((prev) => [...prev, ...parsedMessaged]);
     };
 
-    waku.filter.addEventListener(CONTENT_TOPIC, messageListener);
+    waku.relay.addEventListener(CONTENT_TOPIC, messageListener);
 
     return () => {
-      waku.filter.removeEventListener(CONTENT_TOPIC, messageListener);
+      waku.relay.removeEventListener(CONTENT_TOPIC, messageListener);
     };
   }, [setMessages]);
 
   const onSend = React.useCallback(
     async (nick: string, text: string) => {
-      await waku.lightPush.send({
+      await waku.relay.send({
```

Comment:
is this tested? i get

```
2023-11-07 11:00:21 DBG 2023-11-07 10:00:21.868+00:00 Received error response from handler       tid=1 file=serverprivate.nim:269 status=400 meth=POST peer=*:43734 uri=/relay/v1/messages/%2Fwaku%2F2%2Fdefault-waku%2Fproto error_status="400 Bad Request" error_content_type=text/plain error_message=
```

Reply:
not really, as per this discussion I need cors to be enabled 
https://discord.com/channels/1110799176264056863/1169610038491488267/1169649449669234738